docker-compose.yml

```diff
@@ -1,8 +1,28 @@
 version: "3.9"
 services:
-  wordpress:
-    # Here you put configuration of continer for wordpress that would use image that you build. Ports of webserver should be exposed to lacalhost
-    #...
   database:
-    # Here you put configuration of continer for MySQL that would use any mysql image you like
-    #...
+    image: mysql:8
```

Comment:
why have you removed lines:
```yaml
cap_add:
  - SYS_NICE
 ```
Could you tell me what it does and what does it fix?

Reply:
Before this, I was getting the next warning message on the database service:

```mbind operation not permitted```

As ```cap_add``` allows the management of Linux capabilites, I added SYS_NICE cap because it can handle mbind (Discovered this on StackOverflow when googling the message)

However, I ended removing it as it's more a warning than an error. In other words, it doesn't affect the container's correct behavior.

Reply:
just read what does this option do. What it means for Linux kernel. So next time you will have something in your memory.